src/Pages/Dashboard/Dashboard.jsx

```diff
@@ -51,7 +53,16 @@ class Dashboard extends PureComponent {
         {
           [
             <ChartContainer data={this.props.data} />,
-            <Paper></Paper>,
+            <ShowHistoryGraph
+              state={this.state}
+              selected={this.props.selected}
+              start={this.props.start}
+              end={this.props.end}
+              changeRange={this.props.changeRange}
+              changeCalStart={this.props.changeCalStart}
+              changeCalEnd={this.props.changeCalEnd}
+              historyData={this.props.historyData}
```

Comment:
Looks a bit bulky. Are they necessary?

Reply:
It looks like you probably don't need to pass `state`, but otherwise the other variables are necessary since we're lifting the state up in order to synchronize our data range. [This](https://reactjs.org/docs/lifting-state-up.html) is a pretty neat article about it.